tests/positive/files/hash.go

```diff
@@ -61,6 +64,42 @@ func ValidateFileHashFromDataURL() types.Test {
 	}
 }
 
+func ValidateFileHashFromDataURLForSHA256() types.Test {
+	name := "files.create.withhash.dataurl.sha256"
+	in := types.GetBaseDisk()
+	out := types.GetBaseDisk()
+	config := `{
+	  "ignition": { "version": "$version" },
+	  "storage": {
+	    "files": [{
+	      "path": "/foo/bar",
+	      "contents": {
+			"source": "data:,example%20file%0A",
+			"verification": {"hash": "sha256-352cb4e231c03f9941d54aeee7da755504a7f2096338c609ba5d1b82143419c6"}
+		  }
+	    }]
+	  }
+	}`
+	out[0].Partitions.AddFiles("ROOT", []types.File{
+		{
+			Node: types.Node{
+				Name:      "bar",
+				Directory: "foo",
+			},
+			Contents: "example file\n",
+		},
+	})
+	configMinVersion := "3.0.0"
```

Comment:
This test doesn't actually succeed, does it?  It should only work on `3.1.0-experimental`.